frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.ts

```diff
@@ -51,12 +57,33 @@ export class EETrialWaitingComponent {
   };
 
   constructor(readonly elementRef:ElementRef,
+              readonly cdRef:ChangeDetectorRef,
               readonly I18n:I18nService,
               protected http:HttpClient,
+              readonly Gon:GonService,
               protected notificationsService:NotificationsService,
               public eeTrialService:EnterpriseTrialService) {
   }
 
+  ngOnInit() {
+    let savedDateStr = (this.Gon.get('ee_trial_key') as any).created.split(' ')[0];
+    this.created = new Date(savedDateStr).toLocaleDateString();
+
+    this.eeTrialService.userData$
+      .values$()
+      .pipe(
+        distinctUntilChanged(),
+      )
+      .subscribe(userForm => {
+        this.text.confirmation_info = this.I18n.t('js.admin.enterprise.trial.confirmation_info', {
```

Comment:
Please do not overwrite the whole string, but only the attributes that change.

Reply:
@HDinger I didn't really know how to update specific attributes in the string, but I added a check to avoid the error you described above and fixed the date format.